cmd/ssh.go

```diff
@@ -49,6 +49,10 @@ type SSHCmd struct {
 	GPGAgentForwarding        bool
 	GitSSHSignatureForwarding bool
 
+	// ssh keepalive options
+	SSHKeepAlive         bool          `json:"sshKeepAlive,omitempty"`
```

Comment:
do we need this as an extra bool or could we just set SSHKeepAliveInterval to 0 to disable?